maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -211,8 +211,13 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					Properties frontendProperties;
 					if (server.getVersion().equals(versionsHelper.getLatestSnapshotVersion(artifact))) {
 						frontendProperties = PropertiesUtils.getFrontendPropertiesFromSpaConfigUrl(
-								"https://raw.githubusercontent.com/openmrs/openmrs-distro-referenceapplication/main/frontend/spa-build-config.json");
-					} else {
+								"https://raw.githubusercontent.com/openmrs/openmrs-distro-referenceapplication/main/frontend/spa-assemble-config.json");
+					}
+					else if (new Version(server.getVersion()).higher(new Version("3.0.0-beta.16"))) {
```

Comment:
Do you mind explaining this a bit? For i see both files at: https://github.com/openmrs/openmrs-distro-referenceapplication/tree/main/frontend

Reply:
So here we fetch the frontend modules and add them to the distro.properties. Looks like in newer version those modules were moved to the spa-assemble-config.json file. 

https://github.com/openmrs/openmrs-distro-referenceapplication/commit/158db6182f1b652012755656a1f563ca1d34a900